crates/sui-e2e-tests/tests/per_epoch_config_stress_tests.rs

```diff
@@ -0,0 +1,187 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use move_core_types::language_storage::TypeTag;
+use rand::random;
+use std::path::PathBuf;
+use std::sync::Arc;
+use sui_json_rpc_types::SuiTransactionBlockEffectsAPI;
+use sui_macros::sim_test;
+use sui_types::base_types::{EpochId, ObjectID, ObjectRef, SuiAddress};
+use sui_types::transaction::{CallArg, ObjectArg};
+use sui_types::{SUI_DENY_LIST_OBJECT_ID, SUI_FRAMEWORK_PACKAGE_ID};
+use test_cluster::{TestCluster, TestClusterBuilder};
+
+const DENY_ADDRESS: SuiAddress = SuiAddress::ZERO;
+
+#[ignore]
+#[sim_test]
+async fn per_epoch_config_stress_test() {
+    let test_env = Arc::new(create_test_env().await);
+    let target_epoch = 10;
+    let mut gas_objects = test_env
+        .test_cluster
+        .wallet
+        .get_all_gas_objects_owned_by_address(test_env.regulated_coin_owner)
+        .await
+        .unwrap();
+    let gas1 = gas_objects.pop().unwrap();
+    let gas2 = gas_objects.pop().unwrap();
+    let handle1 = {
+        let test_env = test_env.clone();
+        tokio::spawn(async move { run_transfer_thread(test_env, target_epoch, gas1.0).await })
+    };
+    let handle2 = {
+        let test_env = test_env.clone();
+        tokio::spawn(async move { run_admin_thread(test_env, target_epoch, gas2.0).await })
+    };
+    tokio::try_join!(handle1, handle2).unwrap();
```

Comment:
we might want a timeout - if reconfig halts for unrelated reasons, the test would hang forever

Reply:
also, we may want an assert that at least N transactions were executed by each thread. If i'm reading it correctly, the test could theoretically pass with only 1 transaction executed by each thread.